### PR TITLE
[coregraphics] Update for Xcode 11 beta 1 and 2

### PR DESCRIPTION
**Pull request by spouliot**
along with unit tests for p/invokes

### CODE REVIEW

File: src/CoreGraphics/CGColorConversionInfo.cs [hunk 1]
```diff
@@ -145,14 +145,14 @@ public CGColorConversionInfo (CGColorSpace src, CGColorSpace dst)
...
-				throw new ArgumentNullException (nameof (src));
-			if (dst == null)
-				throw new ArgumentNullException (nameof (dst));
+			if (source == null)
+				throw new ArgumentNullException (nameof (source));
+			if (destination == null)
+				throw new ArgumentNullException (nameof (destination));
 
-			Handle = CGColorConversionInfoCreateWithOptions (src.Handle, dst.Handle, options.GetHandle ());
+			Handle = CGColorConversionInfoCreateWithOptions (source.Handle, destination.Handle, options.GetHandle ());
```
Comment (dalexsoto) [inline src/CoreGraphics/CGColorConversionInfo.cs]: I think there is a potential NRE here

```
Handle = CGColorConversionInfoCreateWithOptions (source.Handle, destination.Handle, options.GetHandle ());
```

`options` can be `null`

Comment (spouliot) [reply src/CoreGraphics/CGColorConversionInfo.cs]: @dalexsoto no, `GetHandle` is an extension method to avoid that :)

Comment (dalexsoto) [reply src/CoreGraphics/CGColorConversionInfo.cs]: Ahh I forgot about it :) thanks!

**Comment by spouliot** [comment]
I'll update it for beta2 before merging it

**Comment by spouliot** [comment]
known issues
* mono-native*/Mac -> https://github.com/xamarin/xamarin-macios/issues/6322
* msbuild -> https://github.com/xamarin/maccore/issues/1266

**Pull request merged**